new-lamassu-admin/src/components/LogsDownloaderPopper.js

```diff
@@ -209,7 +232,9 @@ const LogsDownloaderPopover = ({ name, query, args, title, getLogs }) => {
 
   const radioButtonOptions = [
     { display: 'All logs', code: ALL },
-    { display: 'Date range', code: RANGE }
+    { display: 'Date range', code: RANGE },
+    { display: 'Advanced logs', code: ADVANCED },
+    { display: 'Simplified logs', code: SIMPLIFIED }
   ]
```

Comment:
It's better to create two different arrays for this, no reason to keep this together if we're always slicing it.